### PR TITLE
Users, Tokens, New Config, Patches/Typos

### PR DESCRIPTION
**Pull request by dudeisbrendan03**

### CODE REVIEW

**Review by Superigi — APPROVED** [review]

File: lib/dataHandler.js [hunk 1]
```diff
@@ -41,7 +42,54 @@ lib.create = function (dir, file, data, callback) {
 };
 
 //Read data from a file
-lib.read = functio
```
Comment (Superigi) [inline lib/dataHandler.js]: typo

File: README.md [hunk 2]
```diff
@@ -31,4 +31,4 @@ I have no imports, no need for a package.json or any dependacy management. But t
 - Actually build the authentication part of the API, at the minute it may only deliver content (poorly) and make some simple RESTful requests. I haven't actually put mutch into the API yet.
 
 ### Credits
-No other contributors but me. I'll add people here if they do end up joining in and pushing some changes.
```
Comment (Superigi) [inline README.md:34]: add me

File: config.js [hunk 3]
```diff
@@ -23,7 +23,11 @@ const environments = {
         secured: false, // doesn't need to be secured in local development
         keephttpon: true,
         certloc: './https/cert.pem',
-        keyloc: './https/key.pem'
```
Comment (Superigi) [inline config.js:26]: missed **,**

File: lib/handlers.js [hunk 4]
```diff
@@ -0,0 +1,360 @@
...
+};
+
+
+/*
+ * Users
+ *   post (Create)
+ * Requires: fName, lName, mobile, email, pass, tos
+ * 
+ */
+handlers.user.post = function (data, callback) {
```
Comment (Superigi) [inline lib/handlers.js:38]: Works

File: lib/handlers.js [hunk 5]
```diff
@@ -0,0 +1,360 @@
...
+    }
+};
+
+/*
+ * Users
+ *   get (allow user to access own data)
+ * Requires: email
+ *
+ */
+handlers.user.get = function (data, callback) {
```
Comment (Superigi) [inline lib/handlers.js:90]: Works

File: lib/handlers.js [hunk 6]
```diff
@@ -0,0 +1,360 @@
...
+    }
+};
+
+/*
+ * Users
+ *   put (update user)
+ * Requires: email
+ * Optional: fName, lName, pass, tos (must update one value)
+ */
+handlers.user.put = function (data, callback) {
```
Comment (Superigi) [inline lib/handlers.js:117]: My work that works

**Review by Superigi — APPROVED** [review]

**Review by dudeisbrendan03 — COMMENTED** [review]
Going to run one more test through DevOps

**Comment by dudeisbrendan03** [comment]
Waiting on you

**Pull request merged**